meerket/meerket-application/src/main/java/org/j1p5/api/chat/service/ChatRoomService.java

```diff
@@ -0,0 +1,230 @@
+        Long otherUserId = 2L;
+
+        Map<Long, Integer> unreadCountsMap = Map.of(userId, 0, otherUserId, 0);
+        Map<Long, Boolean> statusMap = Map.of(userId, true, otherUserId, true);
+
+        //TODO 대표이미지, 낙찰가는 아직 product 테이블에 반영x
+        int successfulBid = 100000;
+
+        ProductEntity productEntity = productRepository.findById(productId)
+                .orElseThrow(() -> new IllegalArgumentException("해당 물건이 존재하지않습니다."));
+
+        ChatRoomEntity chatRoomEntity = ChatRoomEntity.create(userId, otherUserId, productId,
+                "productImage", productEntity.getTitle(), successfulBid);
+
+        chatRoomRepository.save(chatRoomEntity);
+
+        CreateChatRoomResponse response = new CreateChatRoomResponse(chatRoomEntity.getId().toString());
+        return response;
+    }
+
+
+    public void exitChatRoom(Long userId, ObjectId roomId) {
+        //TODO 커스텀 예외 처리
+        try {
+            Query query = new Query(Criteria.where("_id").is(roomId));
+            Update update = new Update().set("userStatus." + userId, false);
+
+            mongoTemplate.updateFirst(query, update, ChatRoomEntity.class);
+        } catch (Exception e) {
+            throw new RuntimeException("채팅방에서 나가기 중 예외 발생");
+        }
+    }
+
+
+    public void resetUnreadCount(ObjectId roomObjectId, Long userId) {
+        //TODO 커스텀 예외처리
+        try {
+            Query query = new Query(Criteria.where("_id").is(roomObjectId));
+            Update update = new Update().set("unreadCounts." + userId, 0);
+            UpdateResult result = mongoTemplate.updateFirst(query, update, ChatRoomEntity.class);
+
+            if (result.getMatchedCount() == 0) {
+                throw new IllegalArgumentException("해당 채팅방이 존재하지 않습니다.");
+            }
+        } catch (DataAccessException e) {
+            throw new RuntimeException("안 읽은 메시지 카운트 초기화 중 오류가 발생", e);
+        }
+    }
+
+    public ChatRoomBasicInfo getChatRoomBasicInfo(ObjectId roomObjectId, Long userId) {
+        ChatRoomEntity chatRoomEntity = chatRoomRepository.findById(roomObjectId)
+                .orElseThrow(() -> new IllegalArgumentException("해당 채팅방이 없습니다."));
+
+
+        OtherProfile otherProfile = getOtherProfile(chatRoomEntity, userId);
+        boolean isSeller = chatRoomEntity.getSellerId() == userId;
+
+        return new ChatRoomBasicInfo(
+                chatRoomEntity.getId().toString(),
+                otherProfile.otherNickname(),
+                otherProfile.otherProfileImage(),
+                otherProfile.otherUserId(),
+                chatRoomEntity.getProductId(),
+                chatRoomEntity.getProductTitle(),
+                chatRoomEntity.getProductImage(),
+                chatRoomEntity.getPrice(),
+                isSeller
+        );
+
+    }
+
+
+    // 상대방 프로필 확인
+    private OtherProfile getOtherProfile(ChatRoomEntity chatRoomEntity, Long userId) {
+        Long otherUserId = userId == chatRoomEntity.getSellerId()
```

Comment:
소괄호를 두어 가독성을 증가시키면 좋을 것 같습니다!
```suggestion
        Long otherUserId = (userId == chatRoomEntity.getSellerId()) : ...
```